source/Hazard.cpp

```diff
@@ -60,7 +60,7 @@ void Hazard::Load(const DataNode &node)
 		}
 		else if(key == "environmental effect")
 		{
-			int count = (child.Size() >= 3) ? child.Value(2) : 1;
+			float count = (child.Size() >= 3) ? child.Value(2) : .1;
```

Comment:
`1.`, not `.1`. Also, I think explaining why a count can be a float is in order.
```suggestion
			// Fractional counts may be accepted, since the real count gets multiplied by the strength
			// of the hazard. The resulting real count will then be rounded down to the nearest int
			// to determine the number of effects that appear.
			float count = (child.Size() >= 3) ? child.Value(2) : 1.;
```